WooCommerce/Classes/ViewRelated/Dashboard/Settings/Settings/SettingsViewController.swift

```diff
@@ -192,7 +192,7 @@ private extension SettingsViewController {
     func configureLicenses(cell: BasicTableViewCell) {
         cell.accessoryType = .disclosureIndicator
         cell.selectionStyle = .default
-        cell.textLabel?.text = NSLocalizedString("Open Source Licenses", comment: "Navigates to screen about open source licenses")
+        cell.textLabel?.text = NSLocalizedString("Software Licenses", comment: "Navigates to screen with third party software licenses")
```

Comment:
Same comment as above

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5463/commits/b91fc6e3d0064e4073c5559fe760b998aae4f638